.github/workflows/generate-and-release-openapi-client-ionic.yml

```diff
@@ -0,0 +1,51 @@
+name: generate OpenAPI specification, release if on 'main' branch
+on:
+  workflow_call:
+    inputs:
+      openapi-specification:
+        description: 'OpenAPI specification in YAML format used by the OpenAPI generator'
+        type: string
+        required: true
+      openapi-config:
+        description: 'Configuration file in JSON format used by the OpenAPI generator'
+        type: string
+        default: 'typescript-angular.config.json'
+      base-package-as-path:
+        description: "Value of 'basePackage' inside configuration file converted to a path expression, e.g. 'sam.openapi.api.client' -> 'sam/openapi/api/client'"
+        type: string
+        required: true
+
+jobs:
+  generate-client-typescript-angular:
+    name: client for Typescript/Angular
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Get branch name
+        id: branch-name
+        uses: tj-actions/branch-names@v6.5
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Comment:
In den anderen Workflows hast du überalll v3 auf v4 erhöht, aber in deinen neuen Workflows nicht.
Ist das Absicht?

Reply:
Nein, das ist keine Absicht. Das hätte Dependabot sicher geradegezogen, wenn er es auf `main` entdeckt hätte.